generators/app/templates/package.json

```diff
@@ -64,18 +64,25 @@
     "gulp-sass-glob": "^1.0.6",
     "gulp-sequence": "^0.4.6",
     "gulp-sourcemaps": "^2.1.1",
+    "gulp-concat": "^2.6.1",
     "gulp-stylelint": "^3.7.0",<% if (projectType == 'fe') { %>
     "gulp-twig-up-to-date": "^0.6.1",<% } %>
     "gulp-uglify": "^2.0.0",
     "pre-commit": "^1.2.2",
     "vinyl-buffer": "^1.0.0",
     "vinyl-source-stream": "^1.1.0",
-    "watchify": "^3.7.0"
+    "watchify": "^3.7.0",
+    "browserify-shim": "^3.8.14"
   },
   "dependencies": {
     "normalize-scss": "^6.0.0"<% if (features.has_jquery) { %>,
     "jquery": "^3.1.1"<% } %>
   },
+  "browserify-shim": {
+    <% if (features.has_jquery && has_jquery_vendor_config) { %>
```

Comment:
Is double if necessary? I things it's not possible for has_jquery_vendor_config to be true without features.has_jquery being true.